data/json/monsters/feral_humans.json

```
@@ -215,5 +215,135 @@
     "special_attacks": [ [ "TAZER", 10 ] ],
     "luminance": 500,
     "death_drops": "feral_security_death_drops_flashlight"
+  },
+  {
+    "id": "mon_feral_prepper",
+    "type": "MONSTER",
+    "name": { "str": "feral prepper" },
+    "description": "A quick glance of this person would show that they have prepared well for times like these.  Unfortunately, their bloodshot eyes, and the excluding treatment by the undead shows that they have turned feral.  One of their hands clasp the handle of their prepped improvised weapon, while the other clasps unto the grip of a handgun, treating both as clubs.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 80,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "@",
+    "color": "magenta",
+    "aggression": 75,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 3,
+    "melee_cut": 5,
+    "dodge": 4,
+    "armor_bash": 7,
+    "armor_cut": 7,
+    "armor_bullet": 6,
+    "armor_acid": 4,
+    "vision_day": 50,
+    "vision_night": 3,
+    "harvest": "human",
+    "upgrades": { "into": "mon_feral_survivalist", "half_life": 60 },
+    "starting_ammo": { "9mm": 3 },
+    "special_attacks": [
+      [ "GRAB", 7 ],
+      {
+        "type": "gun",
+        "cooldown": 10,
+        "move_cost": 150,
```

```suggestion
        "move_cost": 150,
```
This means the feral takes 1.5 seconds to aim and shoot.

---

But since they mostly miss their shots, should they pull their trigger more faster, or slower? They treat the gun as a melee weapon in this one for the context, so they shouldn't take time to aim. I haven't thought of this too.

---

Just started experimenting now, sorry for the late I started a bit slacking. I think since they are mostly charging at you, forgetting that their gun is a range weapon, and after a bit of experimenting with debug, they will have a move cost of 50, since they shouldn't take time to aim and miss a lot. (Though, I had a little bit difficult time finding the difference between the 150, 100, and 50 move cost while experimenting, could just be me though)